README.md

```diff
@@ -26,6 +27,7 @@ configure the parameter to use the weekly builds.
 
 **Optional** You can also specify a relative path to the rules file to ignore any alerts from the ZAP scan. Make sure to create
 the rules file inside the relevant repository. The following shows a sample rules file configuration.
+Make sure to use the checkout action(`actions/checkout@v2`) for the repository to download the zap rules files to the ZAP Action.
```

Comment:
`Make sure to checkout the repository (actions/checkout@v2) to provide the ZAP rules to the scan action.`?

Reply:
replaced.